sqlx-macros/src/derives/type.rs

```diff
@@ -88,15 +88,21 @@ fn expand_derive_has_sql_type_transparent(
                     <#ty as ::sqlx::Type<DB>>::compatible(ty)
                 }
             }
-            #[automatically_derived]
-            #[cfg(feature = "postgres")]
```

Comment:
This line was making it so my application had to have a `postgres` feature which was enabled. 💢

Reply:
Woops! Good catch.